packages/apps/purchase/src/components/atoms/RequestButtons/RequestButtons.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import {View, StyleSheet} from 'react-native';
+import {Button, useThemeColor} from '@axelor/aos-mobile-ui';
+import {useSelector, useTranslator, useTypes} from '@axelor/aos-mobile-core';
+
+const RequestButtons = ({}) => {
```

Comment:
This component is not really an atom as it will contain the API logic, maybe more an organisms